docs/source/ontologies_included.md

```diff
@@ -45,7 +35,55 @@ Start creating cuds objects:
 12
 ```
 
-## More ontologies
+## Dublin Core Metadata Initiative (DCMI)
+
+The `dcmitype` and `dcterms` RDFS vocabularies from the [Dublin Core 
+Metadata Initiative (DCMI)](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/)
+can be quickly installed using
+
+```sh
+pico install dcmitype dcterms
+```
+
+Note that due to the fact that OSP-core [does not support RDFS properties](working_with_ontologies.html#limitations), 
+properties in these two vocabularies will be ignored by OSP-core. Only the 
+classes will be detected.
+
+## Data Catalog Vocabulary - Version 2 (DCAT2)
+
+To install the DCAT2 ontology, use
+
+```sh
+pico install dcat2 dcterms foaf prov 
+```
+
+Just installing `dcat2` will fail, as it depends on the `dcterms`, `foaf` 
+and `prov` vocabularies.
```

Comment:
Isn't possible to install dependencies implicitly? Ideally, a user should simply type `pico install dcat2` and the system manages the dependencies. If it's not possible, maybe we can add a statement saying that it is planned to be supported in the future.

Reply:
> > A quick and dirty solution is possible: when installation fails due to missing dependencies, try to install the dependencies too.
> 
> Can you hide this in a `try..catch` block? that would be nice even though not elegant if look what actually happens

Done, see examples [here](https://github.com/simphony/osp-core/pull/760#issuecomment-1077837533). It is not a very simple task due to the fact that one has to recursively resolve the dependencies, both for installation and uninstallation.

Reply:
Nice! I prefer it this way. Only one comment about the logging. Now I see something like this:
```
Additional ontology packages were queued for installation because some of the packages to be installed depend on them: dcmitype was queued because dcterms depends on it; dcterms was queued because dcat2 depends on it; foaf was queued because dcat2 depends on it; prov was queued because dcat2 depends on it
```

Which is very long. Is it possible to have something like:
```
Installing dependencies: dcmitype, dcterms, foaf, prov
```

Reply:
Is [this solution I just committed](https://github.com/simphony/osp-core/pull/760/commits/9e4949d9dcc11045fb106ec88a3b42e086e2d26c) better?

```shell
sh-5.1$ pico install dcat2
INFO 2022-03-25 11:37:13,192 [osp.core.ontology.installation]: Also installing dependencies: foaf, dcterms, prov, dcmitype.
INFO 2022-03-25 11:37:13,192 [osp.core.ontology.installation]: Will install the following namespaces: ['foaf', 'prov', 'dcmitype', 'dcterms', 'dcat2']
INFO 2022-03-25 11:37:13,193 [osp.core.ontology.namespace_registry]: Loading namespaces foaf, http://xmlns.com/foaf/0.1/.
INFO 2022-03-25 11:37:13,193 [osp.core.ontology.parser.owl.parser]: Downloading http://xmlns.com/foaf/spec/index.rdf.
INFO 2022-03-25 11:37:19,447 [osp.core.ontology.namespace_registry]: Loading namespaces prov, http://www.w3.org/ns/prov#.
INFO 2022-03-25 11:37:19,447 [osp.core.ontology.parser.owl.parser]: Downloading http://www.w3.org/ns/prov-o.
INFO 2022-03-25 11:37:20,682 [osp.core.ontology.namespace_registry]: Loading namespaces dcmitype, http://purl.org/dc/dcmitype/.
INFO 2022-03-25 11:37:20,682 [osp.core.ontology.parser.owl.parser]: Downloading https://www.dublincore.org/specifications/dublin-core/dcmi-terms/dublin_core_type.ttl.
INFO 2022-03-25 11:37:21,149 [osp.core.ontology.namespace_registry]: Loading namespaces dcterms, http://purl.org/dc/terms/.
INFO 2022-03-25 11:37:21,150 [osp.core.ontology.parser.owl.parser]: Downloading https://www.dublincore.org/specifications/dublin-core/dcmi-terms/dublin_core_terms.ttl.
WARNING 2022-03-25 11:37:21,773 [osp.core.ontology.parser.owl.parser]: The ontology package dcterms contains the following RDF properties: http://purl.org/dc/terms/accrualPeriodicity, http://purl.org/dc/terms/accrualPolicy, http://purl.org/dc/terms/issued, http://purl.org/dc/terms/modified, http://purl.org/dc/terms/available and 50 more. 
As RDF properties are not supported by OSP-core, the aforementioned properties will be ignored.
INFO 2022-03-25 11:37:21,823 [osp.core.ontology.namespace_registry]: Loading namespaces dcat2, http://www.w3.org/ns/dcat#.
INFO 2022-03-25 11:37:21,823 [osp.core.ontology.parser.owl.parser]: Downloading https://www.w3.org/ns/dcat2.rdf.
INFO 2022-03-25 11:37:23,331 [osp.core.ontology.installation]: Installation successful
```

Reply:
> Is [this solution I just committed](https://github.com/simphony/osp-core/pull/760/commits/9e4949d9dcc11045fb106ec88a3b42e086e2d26c) better?
> 
> ```shell
> sh-5.1$ pico install dcat2
> INFO 2022-03-25 11:37:13,192 [osp.core.ontology.installation]: Also installing dependencies: foaf, dcterms, prov, dcmitype.
> INFO 2022-03-25 11:37:13,192 [osp.core.ontology.installation]: Will install the following namespaces: ['foaf', 'prov', 'dcmitype', 'dcterms', 'dcat2']
> INFO 2022-03-25 11:37:13,193 [osp.core.ontology.namespace_registry]: Loading namespaces foaf, http://xmlns.com/foaf/0.1/.
> INFO 2022-03-25 11:37:13,193 [osp.core.ontology.parser.owl.parser]: Downloading http://xmlns.com/foaf/spec/index.rdf.
> INFO 2022-03-25 11:37:19,447 [osp.core.ontology.namespace_registry]: Loading namespaces prov, http://www.w3.org/ns/prov#.
> INFO 2022-03-25 11:37:19,447 [osp.core.ontology.parser.owl.parser]: Downloading http://www.w3.org/ns/prov-o.
> INFO 2022-03-25 11:37:20,682 [osp.core.ontology.namespace_registry]: Loading namespaces dcmitype, http://purl.org/dc/dcmitype/.
> INFO 2022-03-25 11:37:20,682 [osp.core.ontology.parser.owl.parser]: Downloading https://www.dublincore.org/specifications/dublin-core/dcmi-terms/dublin_core_type.ttl.
> INFO 2022-03-25 11:37:21,149 [osp.core.ontology.namespace_registry]: Loading namespaces dcterms, http://purl.org/dc/terms/.
> INFO 2022-03-25 11:37:21,150 [osp.core.ontology.parser.owl.parser]: Downloading https://www.dublincore.org/specifications/dublin-core/dcmi-terms/dublin_core_terms.ttl.
> WARNING 2022-03-25 11:37:21,773 [osp.core.ontology.parser.owl.parser]: The ontology package dcterms contains the following RDF properties: http://purl.org/dc/terms/accrualPeriodicity, http://purl.org/dc/terms/accrualPolicy, http://purl.org/dc/terms/issued, http://purl.org/dc/terms/modified, http://purl.org/dc/terms/available and 50 more. 
> As RDF properties are not supported by OSP-core, the aforementioned properties will be ignored.
> INFO 2022-03-25 11:37:21,823 [osp.core.ontology.namespace_registry]: Loading namespaces dcat2, http://www.w3.org/ns/dcat#.
> INFO 2022-03-25 11:37:21,823 [osp.core.ontology.parser.owl.parser]: Downloading https://www.w3.org/ns/dcat2.rdf.
> INFO 2022-03-25 11:37:23,331 [osp.core.ontology.installation]: Installation successful
> ```

Note that by doing this you are leaving the user without information on which package specifically depends on which other one (remember that several packages can be installed at once), and chains of dependencies are also flattened.

Reply:
That looks better to me, despite losing some information as you said.